generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -242,19 +252,36 @@ def package_to_apkbuild(ros_distro, package_name,
         depends_export.append(ros_dependency_to_name_ver(dep))
     depends_export_keys = resolve(ros_distro, package_name, depends_export)
 
-    makedepends = []
     catkin = False
     cmake = False
-    for dep in pkg.buildtool_depends:
-        makedepends.append(ros_dependency_to_name_ver(dep))
-        if dep.name == 'catkin':
-            catkin = True
-        elif dep.name == 'cmake':
-            cmake = True
-    if (catkin and cmake) or ((not catkin) and (not cmake)):
-        print('Un-supported buildtool ' + ' '.join(makedepends), file=sys.stderr)
+    ament_cmake = False
+    ament_python = False
+
+    build_type = pkg.get_build_type()
+    if build_type == 'catkin':
+        catkin = True
+    elif build_type == 'cmake':
+        cmake = True
+    elif build_type == 'ament_cmake':
+        ament_cmake = True
+    elif build_type == 'ament_python':
+        ament_python = True
+    else:
+        print('build type is not specified')
         sys.exit(1)
 
+    if not is_ros2:
```

Comment:
[optional] I find it a bit easier to follow writing `if condition` rather than `if not condition` in this case.

Reply:
Done in https://github.com/alpine-ros/ros-abuild-docker/pull/158/commits/73a81d913bc048f24a7dc69eeca2cac7aa8c46cc